### DISCUSSION TITLE
Google Cloud Storage API

### DISCUSSION THREAD

**Discussion by sulmonieri** [davidfrantz/force#69] [Q&A] [answered]
When configuring gsutil to access Google Cloud Level 1 Sentinel-2 Data, a Project ID is requested. Can you confirm that a Project needs to be created on the personal Google account (in this case mine) and this Project needs access to the Bucket gcp-public-data-sentinel-2? However, I have been unsuccessful retrieving an ID of this Bucket. How can I proceed?
I was already able to download Level 1C Data via the Copernicus Hub using Force (force-level1-sentinel2) but as this path was labelled as deprecated, I would like to use force-level1-csd. Note that I am using Ubuntu 16.04 LTS.
Another important reason to use Google Cloud Storage instead of Copernicus is that Data older than that of last year is offline and only accessible by pull request which is limited to one request per user per hour. Theoretically, I could set up a cronjob to set up the requests but as the Google Cloud has all data online I would much prefer this path.

**Comment by davidfrantz** [comment]
Hi @sulmonieri ,

@ernstste can probably shed some light on this, but afaik, you do not need to request the bucket ID.
Our [tutorial ](https://force-eo.readthedocs.io/en/latest/howto/level1-csd.html#tut-l1csd) says: 
``Follow the on-screen instructions to authorize access to the Google Cloud Platform APIs. A project name is not required, just hit enter or make up a name when asked.``

Cheers,
David

**Comment by ernstste** [comment, marked as answer]
The documentation is not 100% accurate here, sorry about that.
I believe you have to enter a project id, but you can simply make one up as it is not needed later in the process.
